tests/unit/s2n_signature_algorithms_test.c

```diff
@@ -469,6 +469,13 @@ int main(int argc, char **argv)
         conn->actual_protocol_version = S2N_TLS13;
         EXPECT_SUCCESS(s2n_connection_set_config(conn, config));
 
+        /* Check that s2n_is_pss_certs_supported() is a superset of s2n_is_rsa_pss_supported() */
+        {
+            if (s2n_is_pss_certs_supported()) {
+                EXPECT_TRUE(s2n_is_rsa_pss_supported());
+            }
+        }
+
```

Comment:
I don't think this test belongs in this file. Can you put it with other tests specifically for rsa_pss?